accepted/random-with-units.md

```diff
@@ -1,4 +1,4 @@
-# Random With Units: Draft 1
```

Comment:
We typically keep the draft indicator even after the specification has been marked accepted, since it's useful to track the version number.